src/controllers/otp.ts

```diff
@@ -37,22 +37,36 @@ export const request = async (req: Request, res: Response) => {
     }).withMessage('message must contains {code} for unique id')
     .trim()
     .run(req);
-  
+
+  await body('callbackUrl')
+    .optional()
+    .matches(/^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$|^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/i).withMessage('invalid format url')
+    .trim()
+    .run(req);
+
+  await body('callbackType')
```

Comment:
create a enums for type and do a validation here
enum OtpCallbackType {
  Simple = 0
}